yass-components-react/src/components/Box/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { ReactNode } from "react";
+
+import * as classNames from "../../../types/yass.json";
+
+export const tags = ["div", "span"];
+
+export interface BoxProps {
```

Comment:
Couple of things about `Box`. 

- I've specifically left `margin` props out as it breaks the component model, by affecting styling beyond the bounds of the component. I've been using `padding` exclusively for a while and haven't run into any problems.
- As for other props, I added what made sense to me, but lemme know if there's something you think should be included.
- I've gone for a pretty 'transparent' implementation for the primitive components. Most props are basically just Yass class names. For example, I opted for:
```
<Box backgroundColor='background-color:blue-500' />

// instead of

<Box backgroundColor='blue-500' />
```

The thinking is:
1. It's easier to type check, since `background-color:blue-500` exists in `../../../types/yass.json`. Since `blue-500` is a substring of that, it's hard to typecheck against. 
2. The purpose of this component library is to kind of show off Yass, so, directly exposing the classnames through props feels like a clearer implementation in terms of displaying what Yass can do.
3. It's worth noting that I don't plan on exposing Yass classes as props for non-primitive components. `<Button variant='primary' />` makes way more sense than `<Button variant='background-color:primary' />`